src/Form/TrickType.php

```diff
@@ -55,27 +55,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'choice_label' => 'name',
                 'label' => 'Groupe'
             ])
-            ->add('cover', TrickMediaImageType::class, [
+            ->add('cover', MediaType::class, [
```

Comment:
You use média type but how you make the difference because heure it's a filtetype only

Reply:
When editing an existing trick, I'm removing this field in the controller and use another form, the CoverType.
If the field isn't removed cover is added with only the 'alt' field since a media exist and it's not a video (Line 82 MediaType.php)

When creating a new trick, I'm going to use an options to add only the fileType in case we use the MediaType form to upload a cover, but it is a part of another branch where I fix the add new trick page based on all the change made in this PR.